tracks/java/exercises/secret-handshake/mentoring.md

```diff
@@ -1,73 +1,80 @@
+### Points to consider in this task
+* binary numeral system understanding
+* Java bitwise operators usage
+* Java enum `ordinal` and `values` methods usage
+
+### Bad practices
+* converting number to binary string and parsing resulting string char by char 
```

Comment:
Move this to common suggestions and how to fix it

Reply:
@MaksimDanilau: Are you still interested in completing this?

Reply:
@MaksimDanilau: Are you still interested in completing this?

Reply:
@sshine No. Time has passed.